pkg/volume/azure_file/azure_file.go

```diff
@@ -139,6 +142,42 @@ func (plugin *azureFilePlugin) newUnmounterInternal(volName string, podUID types
 	}}, nil
 }
 
+func (plugin *azureFilePlugin) RequiresFSResize() bool {
+	return false
```

Comment:
Will not call resize if set to false?

https://github.com/kubernetes/kubernetes/blob/81b43401a9a18e516a2a4de60e0118655d3e8100/pkg/volume/util/operationexecutor/operation_generator.go#L601